numpy/lib/function_base.py

```diff
@@ -2309,7 +2309,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
     else:
         X_T = (X*w).T
     c = dot(X, X_T.conj())
-    c *= 1. / np.float64(fact)
+    c *= np.divide(1., fact)
```

Comment:
Better to use `1` than `1.`, I think - makes the type promotion more lenient.

Reply:
Could also consider `np.reciprocal`

Reply:
I can do that, but I'll need to also change line 2293 to
    
    fact = np.float64(X.shape[1] - ddof)

which could make the rest of this more clear.

Reply:
Sorry, I meant `np.true_divide`

Reply:
done.  Hadn't heard of true_divide before.  Thanks.